ui/src/kapacitor/containers/KapacitorPage.js

```diff
@@ -36,15 +37,22 @@ class KapacitorPage extends Component {
     }
 
     getKapacitor(source, id).then(kapacitor => {
```

Comment:
I made `checkKapacitorConnection` async, but it's not trivial to change this, sadly.